html/objects/_flag.scss

```diff
@@ -7,45 +7,28 @@
 ////
 
 .sprk-o-Flag {
-  display: table;
-  width: 100%;
-}
-
-.sprk-o-Flag__body,
-.sprk-o-Flag__figure {
-  display: table-cell;
-  vertical-align: top;
+  display: flex;
+  align-items: flex-start;
 }
 
 .sprk-o-Flag__figure {
   padding-right: $sprk-flag-gutter;
-
-  > img {
-    display: block;
-    max-width: none;
-  }
 }
 
 .sprk-o-Flag__body {
-  width: 100%;
+  flex: 1;
 
   > :last-child,
   & {
     margin-bottom: 0;
   }
 }
 
-/// Reverses the flag and places the 
+/// Reverses the flag and places the
 /// image on the right of the body content.
 .sprk-o-Flag--rev {
-  direction: rtl;
-
-  > .sprk-o-Flag__body,
-  > .sprk-o-Flag__figure {
-    direction: ltr;
-  }
-
   > .sprk-o-Flag__figure {
+    order: 1;
```

Comment:
What were your thoughts on using the alignment properties vs. using order?

Reply:
I changed it to flex-direction reverse-row -- it feels easier to understand